File: app/src/main/java/ru/artempugachev/popularmovies/data/ReviewResponse.java
package ru.artempugachev.popularmovies.data;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Data class for review response
 */

public class ReviewResponse {
    @SerializedName("id")
    private String id;

    @SerializedName("page")
    private int page;

    @SerializedName("results")
    private List<Review> results;

    public List<Review> getResults() {
        return results;
    }
}

File: app/src/main/java/ru/artempugachev/popularmovies/moviedetails/api/ReviewResponse.kt
package ru.artempugachev.popularmovies.moviedetails.api

import com.google.gson.annotations.SerializedName

import ru.artempugachev.popularmovies.moviedetails.api.Review

/**
 * Data class for review response
 */

data class ReviewResponse (
    @SerializedName("id")
    val id: String,

    @SerializedName("page")
    val page: Int,

    @SerializedName("results")
    val results: List<Review>
)
